Microsoft.Toolkit.Uwp.SampleApp/Assets/mtns.csv

```diff
@@ -1,117 +1,117 @@
-90,Kangpenqing,7281,Baiku Himalaya,28d33m03sN 85d32m44sE,1340,Shisha Pangma,1982,1 (1)
-91,Muztagh Tower,7276,Baltoro Karakoram,35d49m40sN 76d21m40sE,1710,Skil Brum,1956,4 (2)
-92,Mana,7272,Garhwal Himalaya,30d52m50sN 79d36m55sE,730,Kamet,1937,7 (3)
-92,Dhaulagiri VI,7268,Dhaulagiri Himalaya,28d42m31sN 83d16m27sE,485,Dhaulagiri IV,1970,5 (0)
-93,Diran,7266,Rakaposhi-Haramosh Karakoram,36d07m13sN 74d39m42sE,1325,Malubiting,1968,12 (8)
-94,Labuche Kang III/East[12],7250,Labuche Himalaya,28d18m05sN 86d23m02sE,570,Labuche Kang,0,0 (0)
-95,Putha Hiunchuli,7246,Dhaulagiri Himalaya,28d44m52sN 83d08m46sE,1151,Churen Himal,1954,11 (5)
-96,Apsarasas Kangri,7245,Siachen Karakoram,35d32m19sN 77d08m55sE,635,Teram Kangri I,1976,2 (0)
-97,Mukut Parbat,7242,Garhwal Himalaya,30d56m57sN 79d34m12sE,840,Kamet,1951,2 (1)
-98,Rimo III,7233,Rimo Karakoram,35d22m31sN 77d21m42sE,615,Rimo I,1985,1 (0)
-99,Langtang Lirung,7227,Langtang Himalaya,28d15m22sN 85d31m01sE,1525,Shisha Pangma,1978,14 (13)
-100,Karjiang,7221,Kula Kangri Himalaya,28d15m27sN 90d38m49sE,880,Kula Kangri,0,0 (2)
-101,Annapurna Dakshin,7219,Annapurna Himalaya,28d31m06sN 83d48m22sE,775,Annapurna,1964,10 (16)
-102,Khartaphu,7213,Mahalangur Himalaya,28d03m49sN 86d58m39sE,712,Mount Everest,1935,1 (0)
-103,Tongshanjiabu,7207,Lunana Himalaya,28d11m12sN 89d57m27sE,1757,Gangkar Puensum,0,0 (0)
-104,Malangutti Sar,7207,Hispar Karakoram,36d21m47sN 75d08m57sE,515,Distaghil Sar,1985,1 (0)
-105,Noijin Kangsang,7206,Nagarze Himalaya,28d56m48sN 90d10m42sE,2160,Tongshanjiabu,1986,4 (1)
-106,Langtang Ri,7205,Langtang Himalaya,28d22m53sN 85d41m01sE,650,Porong Ri,1981,4 (0)
-107,Kangphu Kang,7204,Lunana Himalaya,28d09m20sN 90d03m48sE,1200,Tongshanjiabu,2002,1 (0)
-108,Singhi Kangri,7202,Siachen Karakoram,35d35m59sN 76d59m01sE,790,Teram Kangri III,1976,2 (0)
-109,Lupghar Sar,7200,Hispar Karakoram,36d21m01sN 75d02m13sE *,730,Momhil Sar,1979,1 (0)
+1,Mount Everest,8848,Mahalangur Himalaya,27d59m17sN 86d55m31sE,8848,none,05/29/1953,>>145 (121)
+2,K2/Qogir,8611,Baltoro Karakoram,35d52m53sN 76d30m48sE,4017,Mount Everest,07/31/1954,45 (44)
+3,Kangchenjunga,8586,Kangchenjunga Himalaya,27d42m12sN 88d08m51sE *,3922,Mount Everest,05/25/1955,38 (24)
+4,Lhotse,8516,Mahalangur Himalaya,27d57m42sN 86d55m59sE,610,Mount Everest,05/18/1956,26 (26)
+5,Makalu,8485,Mahalangur Himalaya,27d53m23sN 87d5m20sE,2386,Mount Everest,05/15/1955,45 (52)
+6,Cho Oyu,8188,Mahalangur Himalaya,28d05m39sN 86d39m39sE,2340,Mount Everest,10/19/1954,79 (28)
+7,Dhaulagiri I,8167,Dhaulagiri Himalaya,28d41m48sN 83d29m35sE,3357,K2,05/13/1960,51 (39)
+8,Manaslu,8163,Manaslu Himalaya,28d33m00sN 84d33m35sE,3092,Cho Oyu,05/9/1956,49 (45)
+9,Nanga Parbat,8126,Nanga Parbat Himalaya,35d14m14sN 74d35m21sE,4608,Dhaulagiri,07/03/1953,52 (67)
+10,Annapurna I,8091,Annapurna Himalaya,28d35m44sN 83d49m13sE,2984,Cho Oyu,06/03/1950,36 (47)
+11,Gasherbrum I,8080,Baltoro Karakoram,35d43m28sN 76d41m47sE,2155,K2,01/01/1958,31 (16)
```

Comment:
I grabbed a couple of the months/dates for testing with the Calendar Picker, but wasn't going to enter them all, so opened https://github.com/CommunityToolkit/WindowsCommunityToolkit/issues/4224 for tracking if a hiking enthusiast wants to help us in the future.